ItalianTranslation_PlanetCoaster.xml

```diff
@@ -12187,14 +12187,14 @@
 		<entry key="HUD_ShopAndFacilityProfits" translation="Negozi &amp; Strutture Profitti"/>
 		<entry key="HUD_ShopMenu_Blueprints" translation="Progetti"/>
 		<entry key="HUD_ShopMenu_Buildings" translation="Strutture"/>
-		<entry key="HUD_ShopMenu_Coasters" translation="Coasters"/>
+		<entry key="HUD_ShopMenu_Coasters" translation="Montagne Russe"/>
 		<entry key="HUD_ShopMenu_Path_Extras" translation="Extra Sentieri"/>
 		<entry key="HUD_ShopMenu_RecentBlueprints" translation="Progetti Recenti"/>
 		<entry key="HUD_ShopMenu_Rides" translation="Attrazioni"/>
 		<entry key="HUD_ShopMenu_Scenery" translation="Scenario"/>
 		<entry key="HUD_ShopMenu_Shops" translation="Negozi e Servizi"/>
 		<entry key="HUD_ShopMenu_Staff" translation="Staff"/>
-		<entry key="HUD_ShopMenu_TrackedRides" translation="Track Rides"/>
+		<entry key="HUD_ShopMenu_TrackedRides" translation="Circuiti"/>
```

Comment:
Qui forse ci penserei un attimo, in quella sezione tra treni e go-kart mi sembra troppo generico "Circuiti"

Reply:
Qualcuno ha tradotto "attrazioni a circuito" ma non volevo ripetere attrazioni che è nel tab successivo.. Come vorresti tradurre?

Reply:
Non saprei devo pensarci, se ti vengono idee metti